test/unittests/libdevcrypto/crypto.cpp

```diff
@@ -181,6 +181,34 @@ BOOST_AUTO_TEST_CASE(verify_secert)
 	BOOST_REQUIRE(k.address());
 }
 
+BOOST_AUTO_TEST_CASE(verifyWithPublicCompressed)
+{
+    PublicCompressed pub{
```

Comment:
Presumably you encrypted the hash into the hex value you're using as the signature value outside of the test case...can we include the encryption as part of the test?

Reply:
It's what `signAndVerify` test does